tests/Java.Interop.Tools.JavaSource-Tests/SourceJavadocToXmldocParserTests.cs

```diff
@@ -168,6 +174,28 @@ @apiSince 1
 </member>",
 				IntelliSenseXml = @"<member>
   <summary>Summary.</summary>
+</member>",
+			},
+			new ParseResult {
+				Javadoc = @"See <a href=""http://man7.org/linux/man-pages/man2/accept.2.html"">accept(2)</a>.  Insert
+more description here.
+How about another link <a href=""http://man7.org/linux/man-pages/man2/accept.2.html"">accept(2)</a>
+@param manifest The value of the <a
+href=""{@docRoot}guide/topics/manifest/manifest-element.html#vcode"">{@code
+android:versionCode}</a> manifest attribute.
+",
+				FullXml = $@"<member>
+  <param name=""manifest"">The value of the <see href=""{DocRootPrefixExpected}guide/topics/manifest/manifest-element.html#vcode""><c>android:versionCode</c></see> manifest attribute.</param>
+  <summary>See <see href=""http://man7.org/linux/man-pages/man2/accept.2.html"">accept(2)</see>.</summary>
+  <remarks>
+    <para>See <see href=""http://man7.org/linux/man-pages/man2/accept.2.html"">accept(2)</see>.  Insert
+more description here.
+How about another link <see href=""http://man7.org/linux/man-pages/man2/accept.2.html"">accept(2)</see></para>
+  </remarks>
+</member>",
+				IntelliSenseXml = $@"<member>
+  <param name=""manifest"">The value of the <see href=""{DocRootPrefixExpected}guide/topics/manifest/manifest-element.html#vcode""><c>android:versionCode</c></see> manifest attribute.</param>
```

Comment:
`<see href="…"/>` is a valid construct?  I wonder when that happened…

Reply:
Looks like it happened in 2017.  Now I feel old. https://github.com/mono/api-doc-tools/commit/fb070470b334d2830556405694c9bb02fd8f4a1c